src/runtime/ExtendedNativeFunctionObject.h

```diff
@@ -0,0 +1,27 @@
+#ifndef __EscargotExtendedNativeFunctionObject__
```

Comment:
ditto.
Others are good to me.

Reply:
Sorry, i totally forgot that, gonna add it in a minute.